js/app/src/api_docs/ApiDocs.svelte

```diff
@@ -148,26 +159,18 @@
 </script>
 
 {#if info}
-	{#if dependencies.filter((dependency) => dependency.show_api).length}
+	{#if api_count}
 		<div class="banner-wrap">
-			<ApiBanner
-				on:close
-				{root}
-				api_count={dependencies.filter((dependency) => dependency.show_api)
-					.length}
-			/>
+			<ApiBanner on:close root={space_id || root} {api_count} />
 		</div>
 		<div class="docs-wrap">
 			<div class="client-doc">
 				<p>
-					Use the <a
-						href="https://gradio.app/docs/#python-client"
```

Comment:
These links were broken but it was never reported because I don't think anyone realized that they were links in the first place. The styling does not make it very clear its a link.